cfgov/unprocessed/apps/analytics-gtm/js/util/analytics-util.js

```diff
@@ -30,7 +30,13 @@ function addEventListenerToElem( elem, event, callback ) {
  * @param {string} msg - Message to load to the console.
  */
 function analyticsLog( ...msg ) {
-  if ( getQueryParameter( 'debug-gtm' ) === true ) {
+
+  // Get query params.
+  const queryParams = new Proxy(new URLSearchParams(window.location.search), {
```

Comment:
What's the point of `Proxy`? Browser support? Why not just do:

```
const queryParams = new URLSearchParams(window.location.search);
// ...
if ( queryParams.get('debug-gtm') === 'true' ) {
```

Reply:
Word is it is faster https://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript#answer-901144

Reply:
FWIW that's only true if you're getting all query params with `getEntries()` instead of individual ones. It's 1.87% faster to just call `.get()`: https://jsben.ch/JGj4X 1.87%!!! 🚀 🚤 💨 💥 

Admittedly, the results slightly change every time I run that benchmark so I trust nothing.

Reply:
I was going to update this to save that sweet sweet 1.87%, but then I had set it to auto-merge and it merged. Maybe I'll make another one later though that removes the Proxy as your solution is simpler to look at.